### ISSUE TITLE
[Bug]: calling requestPermission() returns array with boolean

### ISSUE THREAD

**Issue by Redn4s** [OneSignal/react-native-onesignal#1614]
### What happened?

When requesting the push permission on iOS the response is an array instead of a boolean.

```
OneSignal.Notifications.requestPermission(false).then((accepted) => {
	console.log(accepted); // [true] of [false]
});
```

### Steps to reproduce?

```Markdown
1. Install v5.0.4 of react-native-onesignal
2. Call `requestPermission(false)`
```

### What did you expect to happen?

I expect the `requestPermission` method to return a boolean instead of an array.

### React Native OneSignal SDK version

5.0.4

### Which platform(s) are affected?

- [X] iOS
- [ ] Android

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jennantilla** [comment]
@Redn4s thanks for reaching out! We'll look into this!

**Comment by gpurbia** [comment]
++

**Comment by jennantilla** [comment]
Hello everyone! This is resolved in [Release 5.2.3](https://github.com/OneSignal/react-native-onesignal/releases/tag/5.2.3).

Thanks for your patience!
